features/query/unexpected_arguments/feature.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from temporalio import workflow
+from temporalio.client import RPCError, WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+
+@workflow.defn
+class Workflow:
+    def __init__(self):
+        self.be_done = False
+
+    @workflow.run
+    async def run(self) -> None:
+        await workflow.wait_condition(lambda: self.be_done)
+        return None
+
+    @workflow.query(name="the_query")
+    def the_query(self, arg: int):
+        return f"hi {arg}"
+
+    @workflow.signal(name="finish")
+    def finish_sig(self):
+        self.be_done = True
+
+
+async def checker(_: Runner, handle: WorkflowHandle):
+    # Wrong type TODO: Should be rejected
```

Comment:
Try a string

Reply:
Yeah, it'll work with a string. Keeping this one in so we can see it get fixed tho :)

Reply:
I'm not sure it will get fixed. If Python typing considers `True` an acceptable value for `int`, so do I.